app/src/main/java/openfoodfacts/github/scrachx/openfood/category/mapper/CategoryMapper.java

```diff
@@ -28,6 +28,10 @@ public CategoryMapper() {
                     tag.getProducts()));
         }
         Collections.sort(categories, (first, second) -> first.getName().compareTo(second.getName()));
+        if(categories.get(0).getName().isEmpty()||categories.get(0).getId().isEmpty()){
```

Comment:
Hi @ManavPoddar, what would happen if the categories list is empty? Can you also write some unit tests covering those scenarios?

Reply:
Hi @policorn , Yeah i can do that but i think the above method wont work, as it is not being passed by travis CI build. Maybe have to find another way around, but i'll surely try :)